android/app/src/main/java/org/fossasia/openevent/dbutils/RealmDataRepository.java

```diff
@@ -336,6 +336,7 @@ public Session getSession(String title) {
     }
 
     public RealmResults<Session> getBookMarkedSessionsSync() {
+        Realm realm = Realm.getDefaultInstance();
```

Comment:
Because the updates in the widget are dispatched by another mechanism and this is a synchronous query(non-updating). 

You should not rely on Realm to do it because leaking realm instance can lead to multiple databases being created. App data size can shoot up to 40 MB